src/components/AdjustVault.tsx

```diff
@@ -58,6 +60,12 @@ const AdjustVault = () => {
 
   return (
     <>
+      {isAtRisk && (
+        <div className="text-[#E22951] text-lg mt-4 font-serif font-medium">
+          This vault is at risk. Please increase your collateral or repay your
```

Comment:
"Please" doesn't sound right here. Perhaps "You can". 

Non-blocking as this is a Product matter and trivial to change.

Reply:
prefer to keep it there - we want users to avoid liquidation so I'm in favor of using strong language here. 'You can' doesn't imply urgency the same way as 'Please..